ConsoleMenu/MenuConfig.cs

```diff
@@ -53,7 +53,21 @@ internal MenuConfig(MenuConfig config)
   public Action WriteHeaderAction = () => Console.WriteLine("Pick an option:");
 
   /// <summary>default: (item) => Console.Write("[{0}] {1}", item.Index, item.Name)</summary>
-  public Action<MenuItem> WriteItemAction = item => Console.Write("[{0}] {1}", item.Index, item.Name);
+  public Action<MenuItem> WriteItemAction = item =>
+  {
+    char index;
+
+    if (item.Index >= 10)
```

Comment:
You're right. I missed that. Maybe the play here is to only use this feature when there are >= 10 and <= 36 items, otherwise just use the default.

The other thing we can try is disabling the ability to use the keyboard to individually select a menu item when there are more than 10 or 36, forcing you to use the arrows to navigate.

What do you thing @lechu445?

Reply:
So I fixed the issue where if there are more than 36 items, the indexing reverts back to numerics. But that didn't solve the filtering issue since if you type a letter it's going to try and match it with the actual menu item vs the filter. I'm not sure if there's a way around this.

Reply:
Just pushed the latest changes. The only thing that isn't resolved is when the alphabet is enabled, filtering doesn't work. As I mentioned above, clearer lines would need to be drawn between filter mode and keyboard menu item selection.